src/providers/data_provider_fo.c

```diff
@@ -575,8 +575,13 @@ static void be_resolve_server_done(struct tevent_req *subreq)
     return;
 
 fail:
-    DEBUG(SSSDBG_TRACE_LIBS,
-          "Server resolution failed: [%d]: %s\n", ret, sss_strerror(ret));
+    if (ret == ENOENT) {
+        DEBUG(SSSDBG_TRACE_LIBS,
+              "Server resolution failed[%d]: All servers down\n", ret);
+    } else {
+        DEBUG(SSSDBG_TRACE_LIBS,
+              "Server resolution failed: [%d]: %s\n", ret, sss_strerror(ret));
```

Comment:
In this line we have colon `:` after `failed` and brackets, where in upper line there is one after brackets. You may want to unify this to both have two or single colon.

Reply:
Hi Pawel,

 I have made the necessary changes. Can you please review. 

Thank you!